src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -231,6 +232,7 @@ function WorkspaceInvitePage(props) {
                             onConfirm={inviteUser}
                             showScrollIndicator
                             showLoadingPlaceholder={!didScreenTransitionEnd || !OptionsListUtils.isPersonalDetailsReady(props.personalDetails)}
+                            shouldFocusOnSelectRow={!Browser.isMobile()}
```

Comment:
There are two props - `shouldFocusOnSelectRow` and `shouldPreventDefaultFocusOnSelect`. Can we make it one to make it easy to understand?

Reply:
@MonilBhavsar Can you please check [this discussion](https://github.com/Expensify/App/pull/29084#discussion_r1350309238)?

Reply:
Thanks! Commented there